plugins/disk/quota2percent_

```diff
@@ -0,0 +1,259 @@
+
+  [quota2percent_*]
+  env.warning   [value]        (default: 90)
+  env.critical  [value]        (default: 95)
+  env.language  [en|de|es]     (default: en)
+  env.humanuid  [value]        (default: 1000, only need if there is an other value define for UID_MIN in /etc/login.defs)
+  env.low_uid   [never|no|yes] (default: never)
+                                set to no for producing rrd files for system user, but don't show those graphs (e.g. for later analyses)
+                                if set to yes system user graphs are drawn  
+
+=head1 DESCRIPTION
+
+Wild card Plugin for monitoring the utilization of devices with quota rules. 
+A graph is drawn for each user, which shows the usage as a percentage of his hard limit. System accounts (UID <1000) are suppressed. 
+In addition, a graph is displayed which indicates the ratio device size to device coverage. 
+The script repqutoa, usually part of the package quota, is needed. 
+The plugin itself can be stored in any directory. For example, the device sdb1 shell be monitored, a symbolic link must be created
+in the /etc/munin/plugins/ directory as follows: 
+
+=over  
+
+I<<<  ln -s /<path to file>/quota2percent_  quota2percent_sdb1 >>>
+
+=back
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=head1 VERSION
+
+17.0213
+
+=head1 HISTORY
+
+V17.0213
+
+  fix    some slips
+  add    env.low_uid
+
+V17.0210
+
+  add    env.humanid
+  add    check if device exist
+  add    if no limitations administered via 'quota' for the device the total line is shown only
+  fix    some nitpicking details
+  add    some comments
+
+V17.0131
+
+  add    POD documentation
+  add    env.language
+  add    example graph for Munin Plugin Gallery
+  remove German comments  
+
+V17.0124
+
+  first version, not munin rules conform
+
+=head1 AUTHOR
+
+Jo Hartmann
+
+=head1 LICENSE
+ 
+GPLv2 (L<http://www.gnu.org/licenses/gpl-2.0.html>)
+
+=cut
+
+###################################################
+# Autoconf section                                #
+###################################################
+
+  if [ "$1" = "autoconf" ]; then
+     if ! repquota -V &> /dev/null ; then
+        echo "no ('requota' executable is missing)"
+        exit 0
+     fi
+
+     if ! df "/dev/${0##*_}" &> /dev/null; then
+        echo "no (device /dev/${0##*_} does not exist!)"
+        exit 0
+     fi
+
+     echo yes
+     exit 0
+  fi
+
+###################################################
+# Preparation section                             #
+###################################################
+
+# Load munin's shell libary
+  . "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+# if any fetch from munin-node file
+     Warning=${warning:-90}
+     Critical=${critical:-95}
+     Language=${language:-en}
+     Min_UID=${humanuid:-1000}
+     Low_UID=${low_uid:-never}
+
+# Ensure that the 'root' path is valid
+  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
Are you really sure, that you need to override the path?
This overrules the will of the local admin and I am not sure, if it would work on all platforms.

Since you require the root user anyway, you can just rely on an appropriately configured PATH, I guess.